assets/src/edit-story/output/page.js

```diff
@@ -67,12 +80,20 @@ function OutputPage({ page }) {
           ))}
         </div>
       </amp-story-grid-layer>
+      {ctaElements.length && index >= 1 && (
```

Comment:
I think the `index >= 1` rule should be enforced on the data model level, not here.